clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -2483,7 +2509,7 @@ RValue CodeGenFunction::EmitBuiltinExpr(const GlobalDecl GD, unsigned BuiltinID,
     case Builtin::BI__builtin_sinf128:
       return RValue::get(emitUnaryMaybeConstrainedFPBuiltin(*this, E,
                                    Intrinsic::sin,
-                                   Intrinsic::experimental_constrained_sin));
+                                   Intrinsic::fpbuiltin_sin));
```

Comment:
Currently this patch is only handling sin, cos, tan, and sincos, right? We'll need to add support for the other functions in the fpbuiltin table at some point.

Reply:
Yes. Will do.